spec/features/admin_process/admin_reward_process/admin_create_reward_spec.rb

```diff
@@ -0,0 +1,101 @@
+require 'rails_helper'
+require 'capybara/rspec'
+
+RSpec.describe 'Admin create reward', type: :feature do
+  let(:admin) { create(:admin) }
+  let(:reward) { create(:reward) }
+
+  before do
+    visit root_path
+  end
+
+  it 'Create Reward with correct credentials' do
+    click_link 'Sign in as Admin'
+    fill_in 'Email', with: admin.email
+    fill_in 'Password', with: admin.password
+    click_button 'Log in'
+    expect(page).to have_current_path(admins_root_path)
+    click_link 'Manage Rewards'
+    expect(page).to have_current_path(admins_rewards_path)
+    click_link 'New Reward'
+    fill_in 'Title', with: reward.title
+    fill_in 'Description', with: reward.description
+    fill_in 'Price', with: reward.price
+    click_button 'Create Reward'
+    expect(page).to have_content 'Reward was successfully created.'
+    expect(page).to have_current_path(admins_rewards_path)
+  end
+
+  it 'Create Reward without a title' do
```

Comment:
not an error, but you have already covered all validations in the model spec - there is no need to test it again in the system spec (which takes longer)

Reply:
I just wanted to check content about validations.